OMX-27-firmware/OMX-27-firmware.ino

```diff
@@ -819,36 +900,36 @@ void dispGenericMode(int submode, int selected){
 
 
 	switch(selected){
-		case 0:
+		case 1:
 			display.fillRect(0*32+2, 9, 29, 21, WHITE);
 			break;
-		case 1: 	//
+		case 2: 	//
 			display.fillRect(1*32+2, 9, 29, 21, WHITE);
 			break;
-		case 2: 	//
+		case 3: 	//
 			display.fillRect(2*32+2, 9, 29, 21, WHITE);
 			break;
-		case 3: 	//
+		case 4: 	//
 			display.fillRect(3*32+2, 9, 29, 21, WHITE);
 			break;
-		case 4: 	//
+		case 0: 	//
 			break;
 		default:
 			break;
 	}
 	// ValueBoxes
 	int highlight = false;
-	for (int j = 0; j<4; j++){
+	for (int j = 1; j<5; j++){
```

Comment:
5 is hardcoded in a lot of places, you should swap the magic number with `#define NUM_PARAMS 5` so that it's more obvious what this number is for. Also starting at 1 is a little weird, makes me wonder if there's an off-by-one error somewhere.

Reply:
see previous comment , but yeah a num_params would be better.

Reply:
added `NUM_DISP_PARAMS` [below](https://github.com/okyeron/OMX-27/pull/28/commits/c8549e763622fd6fe94a175216426f35c34852e2)